### PR TITLE
feat: add light theme

### PR DESCRIPTION
**Pull request by aslilac**
Screenshot 2023-12-14 at 3 46 31 PM
Screenshot 2023-12-14 at 3 46 43 PM

### CODE REVIEW

File: site/package.json [hunk 1]
```diff
@@ -104,6 +104,7 @@
     "@storybook/addon-essentials": "7.5.2",
     "@storybook/addon-links": "7.5.2",
     "@storybook/addon-mdx-gfm": "7.5.2",
+    "@storybook/addon-themes": "^7.6.4",
```
Comment (BrunoQuaresma) [inline site/package.json:107]: Nit: Remove the `^`

**Review by BrunoQuaresma — APPROVED** [review]
Overall the code looks good! Is there any place you think we should be more careful in the review? I'm going to run it locally to do some QA.

**Review by BrunoQuaresma — COMMENTED** [review]
I found a few more issues:

1. The loading spinner in the workspace status badge and template editor build status badge are white.

Screenshot 2023-12-19 at 10 52 34

2. The contrast of the agent metadata is low
Screenshot 2023-12-19 at 10 49 34

3. The icon of the GitHub button is white
Screenshot 2023-12-19 at 10 48 58

4. The default icons in the app usage panel, on the template insights page are white
Screenshot 2023-12-19 at 10 48 41

5. The template source files theme colors have a low contrast
Screenshot 2023-12-19 at 10 48 09

6. The role badge, on the users page has a dark text on a dark background
Screenshot 2023-12-19 at 10 47 25

7. When a pagination item is active, it has a dark text on dark background
Screenshot 2023-12-19 at 10 47 06

8. The menu item on hover does not apply a different/contrast background
Screenshot 2023-12-19 at 10 46 19

9. The help tooltip links have a low contrast
Screenshot 2023-12-19 at 10 46 09

10. The rocket icon in the deployment bottom bar is white
Screenshot 2023-12-19 at 10 45 48

**Comment by bpmct** [comment]
Looking awesome! Some feedback:

---

The contrast seems too light here

image

in comparison here's the contrast for the dark theme

image

---

Same with the create workspace button IMO

image

versus

image

in this case, it felt like I was "discovering" the button in the light mode (like where's waldo) where my eyes were drawn to it in the dark one

--- 

Should this be red?

image

--- 

should this be green?

image

---

should this be red?

image

---

this blue is a bit jarring. perhaps if the "create workspace" was blue it wouldn't be

image

---

audit log diffs seem quite jarring too since the rest of ui has minimal constrast

image

---

**Comment by matifali** [comment]
Users Table
Screenshot 2023-12-19 at 7 33 39 PM
Groups:
image

**Comment by matifali** [comment]
I suggest we make the Green more greener. It will help improve the contrast for agent metadata and the health page.
Screenshot 2023-12-19 at 7 37 10 PM

**Pull request merged**